src/components/DataViewFilterMultiselect/DataViewFilterMultiselectTypes.ts

```diff
@@ -27,4 +28,10 @@ export interface DataViewFilterMultiselectDropdownContentProps {
 	onApply: (val: any) => void;
 	isOpen: boolean;
 	placeholder?: DataViewFilterMultiselectProps["args"]["placeholder"];
+	limit?:  DataViewFilterMultiselectProps["args"]["limit"];
+	onChange?: (val: any) => void;
+	hideButtons?: boolean;
+	createNewOption?: (val: any) => void;
```

Comment:
Create new option returns a MosaicLabelValue and should be a promise because we should wait until it gets inserted in the DB. (the same way as AdvancedSelectionInputSettings does.